### ISSUE TITLE
Unknown string value class: class com.jetbrains.python.psi.impl.PyParenthesizedExpressionImpl

### ISSUE THREAD

**Issue by gitmehedi** [wt-io-it/odoo-pycharm-plugin#24]
Plugin Version: 0.5.12
IDEA Version: PyCharm 2021.3.3

<details><summary>java.lang.Throwable: Unknown string value class: class com.jetbrains.python.psi.impl.PyParenthesizedExpressionImpl</summary>

```
java.lang.Throwable: Unknown string value class: class com.jetbrains.python.psi.impl.PyParenthesizedExpressionImpl
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:182)
	at at.wtioit.intellij.plugins.odoo.models.OdooModelUtil.getStringValueForValueChild(OdooModelUtil.java:104)
	at at.wtioit.intellij.plugins.odoo.models.OdooModelUtil.detectName(OdooModelUtil.java:42)
	at at.wtioit.intellij.plugins.odoo.models.OdooModelUtil.detectName(OdooModelUtil.java:28)
	at at.wtioit.intellij.plugins.odoo.models.index.OdooModelDefinition.<init>(OdooModelDefinition.java:17)
	at at.wtioit.intellij.plugins.odoo.models.index.OdooModelFileIndex$OdooModelFileIndexer.lambda$mapWatched$0(OdooModelFileIndex.java:114)
	at at.wtioit.intellij.plugins.odoo.PsiElementsUtil.walkTree(PsiElementsUtil.java:50)
	at at.wtioit.intellij.plugins.odoo.models.index.OdooModelFileIndex$OdooModelFileIndexer.mapWatched(OdooModelFileIndex.java:111)
	at at.wtioit.intellij.plugins.odoo.index.OdooDataIndexer.lambda$map$0(OdooDataIndexer.java:13)
	at at.wtioit.intellij.plugins.odoo.index.IndexWatcher.runIndexJob(IndexWatcher.java:14)
	at at.wtioit.intellij.plugins.odoo.index.OdooDataIndexer.map(OdooDataIndexer.java:13)
	at at.wtioit.intellij.plugins.odoo.index.OdooDataIndexer.map(OdooDataIndexer.java:9)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapByIndexer(MapReduceIndex.java:306)
	...
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:214)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	...
	at java.base/java.lang.Thread.run(Thread.java:829)
```
</details>

**Comment by ap-wtioit** [comment]
@gitmehedi thanks for reporting this. Could you check if Find in Files `[CTRL]+[SHIFT]+[F]` with the following regex finds anything and report the (anonymized) results back to me here: `\s_(name|inherit)\s*=\s*\(`

This would help me writing a better fix for this.

**Comment by ap-wtioit** [comment]
Fixed at least a version of the error in cc4e37eab3a3356b787dbc5346d6d4943c697681:
```python
class MyModel(models.BaseModel):
    _name = ("my_module.my_model")
```
When there is an extraneous parentheses around the name/inherit value.
